apisix/cli/ops.lua

```diff
@@ -279,6 +279,12 @@ Please modify "admin_key" in conf/config.yaml .
         end
     end
 
+    if yaml_conf.apisix.dns_resolver_valid then
+        if tonumber(yaml_conf.apisix.dns_resolver_valid) == nil then
```

Comment:
Can we use `JSON Schema` to check configuration values?

Reply:
Seems that the whole `config.yaml` doesn't have the json schema.

Reply:
I think it is time to do this in this way now

Reply:
It's more proper to do it in another PR.